Dockerfile

```diff
@@ -25,4 +25,4 @@ RUN pip install transformers==4.21.0 \
                 matplotlib==3.6.3 \ 
                 seaborn==0.12.2
 
-USER $DOCKER_UID:$DOCKER_GID
\ No newline at end of file
+USER $DOCKER_UID:$DOCKER_GID
```

Comment:
In most cases, `DOCKER_UID` and `DOCKER_GID` should be passed from the parent process via the environment.